bindings/core/src/response.rs

```diff
@@ -247,6 +247,7 @@ pub enum Response {
     Output(Output),
     /// Response for:
     /// - [`AccountIdToBech32`](crate::method::ClientMethod::AccountIdToBech32)
+    /// - [`AnchorIdToBech32`](crate::method::ClientMethod::AnchorIdToBech32)
```

Comment:
Looks like we're missing the utils methods here?

Reply:
![image](https://github.com/iotaledger/iota-sdk/assets/3305068/1160c0ef-3e4f-45e5-9829-59263d229835)

Reply:
Sorry, but was my last comment for this PR :trollface:

Reply:
Done